js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,12 @@
+function freqSort(arrayOfString) {
+  let frequency = {};
+  arrayOfString.map((string) => {
+    frequency.hasOwnProperty(string)
+      ? (frequency[string] = frequency[string] + 1)
+      : (frequency[string] = 1);
+  });
```

Comment:
code for generating frequency can be split up into a different function

Reply:
also use of map function for calculating frequency is wrong. Instead you should have used for of loop.
And  one more point to highlight that you are not even returning anything from the map predicate function so why even use it in the first place.

Important Note: Map function is not a general purpose looping function